index.js

```diff
@@ -51,6 +51,19 @@ module.exports = {
       fs.unlink(path.join(testAppPath, '.travis.yml')),
     ];
 
+    if (options.vitest) {
+      tasks.push(
+        (async () => {
+          // lookupBlueprint (used above) does not lookup packages on npm
+          await execa(
```

Comment:
should we do this same technique for the app as well? 
it would allow us to add ember-cli-typescript for TS porjects.